### PR TITLE
unset neutral settings

### PR DESCRIPTION
**Pull request by MaxBittker**
committed_deploys_only is the default setting, and so instead of ever saving it, it's better to unset. this means I had to change unset_value to be kwargs to also accept organization.

### CODE REVIEW

File: src/sentry/models/useroption.py [hunk 1]
```diff
@@ -68,13 +68,17 @@ def get_value(self, user, key, default=None, **kwargs):
             result = self.get_all_values(user, project)
         return result.get(key, default)
 
-    def unset_value(self, user, project, key):
-        # this isn't implemented for user-organization scoped options yet, because it hasn't been needed
-        self.filter(user=user, project=project, key=key).delete()
+    def unset_value(self, user, key, **kwargs):
```
Comment (mattrobenolt) [inline src/sentry/models/useroption.py]: Definitely need a test for this behavior.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Review by mattrobenolt — CHANGES_REQUESTED** [review]
plz run your code before committing next time

File: tests/sentry/models/test_useroption.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        self.org = self.create_organization(owner=None)
+        self.team = self.create_team(organization=self.org)
+        self.create_member(user=self.user, organization=self.org, teams=[self.team])
+
+        UserOption.objects.set_value(
+            user=self.user,
+            organization=self.org,
+            key='deploy-emails',
+            value=UserOptionValue.all_deploys,
+        )
```
Comment (mattrobenolt) [inline tests/sentry/models/test_useroption.py]: You should add an assert here to make sure that the `set_value` worked correctly.

**Comment by MaxBittker** [comment]
might remove the code logic changes and just commit some form of this test

**Pull request closed without merging**